FlowCrypt/src/main/java/com/flowcrypt/email/database/entity/MessageEntity.kt

```diff
@@ -251,13 +260,17 @@ data class MessageEntity(
     result = 31 * result + (threadId?.hashCode() ?: 0)
     result = 31 * result + (historyId?.hashCode() ?: 0)
     result = 31 * result + (password?.contentHashCode() ?: 0)
+    result = 31 * result + (draftId?.hashCode() ?: 0)
     result = 31 * result + from.hashCode()
     result = 31 * result + replyToAddress.hashCode()
     result = 31 * result + to.hashCode()
     result = 31 * result + cc.hashCode()
     result = 31 * result + msgState.hashCode()
     result = 31 * result + isSeen.hashCode()
+    result = 31 * result + isDraft.hashCode()
+    result = 31 * result + isOutboxMsg.hashCode()
     result = 31 * result + uidAsHEX.hashCode()
+    result = 31 * result + isPasswordProtected.hashCode()
```

Comment:
What `31` value means here? Maybe it'll be better to move it to separate variable (variable name will help to understand what it means)?

Reply:
I think it's a way to combine the various hash codes of the underlying data into one hash code, a bit like XORing binary values, but not really. We just learn to ignore these when reading Kotlin code :-)

Reply:
Ah, I see, didn't notice that it was auto-generated 👍

Reply:
Android Studio generates `equals() `and `hashCode()` when we need it. It will be better don't change these methods manually.